wpilib/__init__.py

```diff
@@ -1,17 +1,24 @@
-from . import _init_wpilib
-
 # TODO: robotpy-build subpackage bug
 from wpimath._controls._controls import trajectory as _
 
+from . import _init_wpilib
+from ._impl.main import run
+
 # autogenerated by 'robotpy-build create-imports wpilib'
 from ._wpilib import (
     ADIS16448_IMU,
     ADIS16470_IMU,
     ADXL345_I2C,
     ADXL345_SPI,
     ADXL362,
-    ADXRS450_Gyro,
+    CAN,
```

Comment:
Why is this sort not alphabetical? That's just weird.

Reply:
huh isort is supposed to be alphabetical, that is weird.

Reply:
My guess is `ALL_CAPS` > `MixedCaps` > `all_lower`